lib/utils.js

```diff
@@ -219,8 +218,13 @@ var Utils = module.exports = {
   },
 
   toDefaultValue: function(value) {
-    if (lodash.isFunction(value)) {
-      return value();
+    if (typeof value === 'function') {
+      var tmp = value();
+      if (tmp instanceof DataTypes.ABSTRACT) {
```

Comment:
Defining a model should be calling `new` on it, so we have to handle both cases.